### ISSUE TITLE
GSE130973 - Aging Human skin

### ISSUE THREAD

**Issue by mshadbolt** [ebi-ait/hca-ebi-wrangler-central#107]
**Project label**
scAgingHumanMaleSkin

**Primary Wrangler:**
Marion & Ami

**Secondary Wrangler:**
Ray

**Associated files:**

Google Drive:
https://docs.google.com/spreadsheets/d/1qhGETqKS5PPg-AVluaUiac6kaivNsLsk/edit#gid=758716052

**Published study links**

Paper:
https://www.nature.com/articles/s42003-020-0922-4
https://www.biorxiv.org/content/10.1101/633131v1.full

Accessioned data:
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE130973

[**Ingest**](https://contribute.data.humancellatlas.org/projects/detail?uuid=51f02950-ee25-4f4b-8d07-59aa99bb3498)

**Key Events**

- [x] convert published metadata to HCA spreadsheet
- [x] manually curate dataset to meet HCA metadata standard
- [x] Upload sheet to validate metadata
- [x] Check linking using ingest graph validator
- [x] Transfer files to ingest to validate data files
- [x] Ask the Secondary Wrangler for an end-to-end review of the project. Ask the Expertise Wrangler to review specific tabs if needed
- [x] Submit dataset to Production

**Comment by mshadbolt** [comment]
I was able to almost fully wrangle this dataset, the issues are:
* Dissociation incorporates both mechanical and enzymatic dissociation but our schema only allows for one method to be specified. I created a metadata schema ticket to change this to an array but given it is unlikely to be actioned any time soon I used the more generic 'sample dissociation' term and put mechanical and enzymatic in the text field
* The GEO dataset and paper are inconsistent in terms of what sequencer was used, I emailed the authors but I will go with what it says in GEO if I don't hear back.

I am currently transferring files from ENA into an hca-util upload area
The spreadsheet [passed ingest](https://ui.ingest.archive.data.humancellatlas.org/submissions/detail?id=5f2933a04f36d156a8bd92f3&project=3138bf07-b24a-49c8-b1f3-61329d7abc3b) and graph validation
@ami-day are you able to do a secondary review during the upcoming sprint?

**Comment by mshadbolt** [comment]
@rays22 would you also be able to review this one? it is pretty small and simple

**Comment by rays22** [comment]
Secondary review
----------------

* HCA metadata spreadsheet [GSE130973.xlsx](https://drive.google.com/file/d/10OtiVeOs9rVg2APP3gnT-qQ3awlIG4Sz/view?usp=sharing)
**passed** validation by the `ingest-graph-validator`. However, Ingest submission [Aug 4, 2020, 11:08:32 AM	e984c0a8-7a8d-4a23-ac2a-596b3ab2b128](https://contribute.data.humancellatlas.org/submissions/detail?uuid=e984c0a8-7a8d-4a23-ac2a-596b3ab2b128&project=3138bf07-b24a-49c8-b1f3-61329d7abc3b)
failed to load by the `ingest-graph-validator` into a database, which could be due to a bug in the tool or I may have been using the wrong API end-point, or something else.
* The experimental design in the spreadsheet **looks OK** to me. I have uploaded the experimental design graph `GSE130973_graph2.svg` to the Google drive folder of this dataset. The experimental graph is shown without any protocols to make it clearer.
* The data files look valid in Ingest UI.

* Some of the possible ontology terms are **missing** in the metadata spreadsheet.
e.g: `LIBRARY CONSTRUCTION METHOD ONTOLOGY ID LIBRARY CONSTRUCTION METHOD ONTOLOGY LABEL ONTOLOGY ID`

@mshadbolt ,
Please, let me know if you would like me to fill in the missing ontology terms.

**Comment by mshadbolt** [comment]
Thanks @rays22 I forgot to run it through the ontology filler script

Would you agree that the inguinal region they took their skin samples from would be best described by the ontology term `skin of pelvis`? https://ontology.staging.archive.data.humancellatlas.org/ontologies/hcao/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0001415

**Comment by rays22** [comment]
For *'the sun-protected inguinoiliac region'* in the paper the term `inguinal part of abdomen (UBERON:0008337)` looks correct to me. I think the pelvis region would be more lateral relative to the [inguinal region](https://i.warosu.org/data/sci/img/0090/27/1499735236484.jpg), but not very far off.
I guess you would like the most specific term to be part of `zone of skin`, but the term `skin of the inguinal part of abdomen` does not exist in the ontology yet.
In that case, the more generic `abdominal segment skin UBERON:0003836` might be another option?

I have run the spreadsheet through the ontology filler, and it added these terms/labels:

* `Library preparation protocol` tab:
Found exact match for term polyA RNA (Ontology: OBI:0000869)
Found exact match for term 10x 3' v2 sequencing (Ontology: EFO:0009899)
* `Sequencing protocol` tab:
Found exact match for term tag based single cell RNA sequencing (Ontology: EFO:0008440)
Found exact match for term tag based single cell RNA sequencing (Ontology: EFO:0008440)

I see that you have already run the ontology filler too, so I am not going to upload my version of the spreadsheet.

**Comment by mshadbolt** [comment]
This is ready for export when we are able to export again. Should also be suitable for SCEA

**Comment by aaclan-ebi** [comment]
Hi, just want to note that this submission was affected by the production data files deletion incident which can be tracked in [this ticket](ebi-ait/hca-ebi-dev-team/383),  the data files for this dataset needs to be reuploaded before we can submit it.

**Comment by ami-day** [comment]
I am moving this ticket to done as it is tagged with gdpr. We can't curate to SCEA without paths to fastq or bam files or sra objects.

**Comment by ami-day** [comment]
Oh wait, looks like the data files might be missing in ingest prod.?

**Comment by ami-day** [comment]
-Asked Algeria about deleted files. Are they still missing?
-Asked Tony about potential data privacy issues - can we upload the fastq to dcp

**Comment by aaclan-ebi** [comment]
@ami-day unfortunately, the files here haven't been restored yet. :( Let me know if i can help with reuploading the files. I am not sure if the files are in the hca-util upload area. You may have to download/ask the contributor again for the files. Apologies for the inconvenience.

**Comment by clairerye** [comment]
Can we not submit it with matrices but no fastq files if it is subject to GDPR?

**Comment by ami-day** [comment]
> Can we not submit it with matrices but no fastq files if it is subject to GDPR?

Yep, at the time we couldn't submit matrices only, but now we can, so I'll have a go at this.

**Comment by ami-day** [comment]
Updated the sheet to submit matrix files instead of sequence files and have submitted the project: https://contribute.data.humancellatlas.org/submissions/detail?id=60cb1648e259f076612626a3 it is currently exporting.

**Comment by ami-day** [comment]
exported and submitted import form.

**Comment by idazucchi** [comment]
It should be techincally possible to add the fastq files with an update but this task is low priority for two reasons:
1. the skin network is not particualrly interested in fastq files
1. we are giving priority to the first 6 draft atlases - our help is most needed for gut, brain and eyes

**Comment by ofanobilbao** [comment]
Re-opening for investigation as per the Ops meeting last week

**Comment by ofanobilbao** [comment]
@idazucchi can you check if the FASTQ can be shared considering it has the GDPR label? If not, we should close this ticket. If yes, let's move it to Needs Update and proceed

**Comment by ofanobilbao** [comment]
@idazucchi have you managed to investigate if it's possible to add the FASTQ considering it has the GDPR label? So that we can either close the ticket or move it to the Needs Update column?

**Comment by idazucchi** [comment]
Hi @ofanobilbao :)
We _can_ add fastq files to the project
1. When living donor datasets already have sequence data publicly available we are allowed to publish the same data in the DCP
2. a previous comment from Ami says "-Asked Tony about potential data privacy issues - can we upload the fastq to dcp"

I'll work on this before starting a new project but it will be blocked at the file validation stage like the rest of the datasets

**Comment by idazucchi** [comment]
I've added the fastq files - can I just export it?

**Comment by idazucchi** [comment]
Files added to the browser!
